torch/csrc/jit/runtime/static/impl.h

```diff
@@ -151,6 +163,25 @@ class TORCH_API StaticRuntime {
     DCHECK(i < module_->output_regs.size());
     return reg_[module_->output_regs[i]];
   }
+  friend class MemoryPlanner;
```

Comment:
let's avoid this by providing `nodes() const` and `inference_module() const` functions